### ISSUE TITLE
Cannot find module 'solid-jotai/utils' or its corresponding type declarations

### ISSUE THREAD

**Issue by rtritto** [wobsoriano/solid-jotai#2]
```ts
import { useHydrateAtoms } from 'solid-jotai/utils'
```

`tsconfig.json`
```json
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "esModuleInterop": true,
    "isolatedModules": true,
    "lib": [
      "DOM",
      "DOM.Iterable",
      "ESNext"
    ],
    "jsx": "preserve",
    "jsxImportSource": "solid-js",
    "module": "ESNext",
    "moduleResolution": "bundler",
    "noEmit": true,
    "skipLibCheck": true,
    "strict": true,
    "target": "ESNext",
    "types": [
      "vite/client"
    ]
  }
}
```

### Versions:
jotai: _2.0.4_
solid-jotai: _0.2.0_
typescript: _5.0.4_

### Console:
```console
Missing "./utils" specifier in "solid-jotai" package
```

**Comment by wobsoriano** [comment]
Thanks. Please try 0.2.1.

**Comment by rtritto** [comment]
I get same errors

**Comment by RIP21** [comment]
@wobsoriano you have failing CI builds with typechecking failing. Maybe that's the reason? :)

**Comment by wobsoriano** [comment]
@RIP21 the failing CI build is from the test files. Im gonna look at it again in a bit :)

**Comment by rtritto** [comment]
FYI @dai-shi

https://github.com/wobsoriano/solid-jotai/commit/96e366844e82b0355669c1249456edf03f831e1c is the first fix.

**Comment by wobsoriano** [comment]
Fixed in 0.2.2.

**Comment by rtritto** [comment]
Thanks!
